luigi/contrib/hive.py

```diff
@@ -543,7 +514,22 @@ def path(self):
         return location
 
     def open(self, mode):
```

Comment:
actually I think we don't need to declare this method at all as it's not present in base class `luigi.Target` but it's present in 'FilySystemTarget' only
@Tarrasch @dlstadther @honnix 
WDYT?